custom_manifests/manifests/04-cluster-network.yml

```diff
@@ -0,0 +1,17 @@
+# ----- 04-cluster-network.yml -----
+
+# required for iscsi boot volumes on OCP 4.17+
```

Comment:
Doing so would require making the intended OCP version a variable in the terraform. I did implement this, but we opted for including all the manifests since there wasn't any harm in doing so and we wouldn't need to also maintain a list of OpenShift versions.